lib/submission_runners/ruby.rb

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+require 'submission_runners/base'
+
+module SubmissionRunners
+  class Ruby < Base
+    def self.image
+      'ruby:2.4.1'
+    end
+
+    def initialize(*args)
+      @ruby_builder = Docker::RubyBuilder.new(self)
+      super
+    end
+
+    delegate :build, :run, to: :@ruby_builder
+
+    def container_user
+      "root"
```

Comment:
Do not do this. We purposefully chose the "nobody" user because it has very limited permissions. Running unknown code in production under the root user is a massive security risk.